scripts/wheels/linux/build.sh

```diff
@@ -2,13 +2,29 @@
 # PYTHONS=("cp36" "cp37" "cp38" "cp39" "cp310")
 PYTHONS=("cp38")
 export KRATOS_VERSION="9.0.0"
+export KRATOS_BUILD_TYPE="Custom"
+
+export HASH=$(git show -s --format=%h) # Used in version number
+if [[ "${KRATOS_BUILD_TYPE}" != "Release" ]]; then
+    export KRATOS_VERSION="${KRATOS_VERSION}.dev.${HASH}"
```

Comment:
alternatives e.g. `git describe --tag` -> `v9.0-<commits since tag>-<git hash> or also many python packages exist for this.

Reply:
including `dev` in the version name explicitly distinguishes from the actual release and is e.g. not installed using standard `pip install` command - requires  `pip install --pre`